### PR TITLE
Change application configuration for new log configuration struct

### PR DESCRIPTION
**Pull request by kaplanelad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +74,9 @@ Parameter | Description | Default
 | **Watcher** |
 `watcher.kubernetes.create` | If true, Kubernetes watcher will be deploy | `true`
 `watcher.kubernetes.annotations` | The annotations used in api deployment | `{}`
-`watcher.kubernetes.application.log_level` | The application log level | `info`
+`watcher.kubernetes.application.log.level` | The application log level | `info`
+`watcher.kubernetes.application.log.gelf_address` | The address for ship the log out for external system | undefined
+`watcher.kubernetes.application.log.level` | The application log level | `info`
```
Comment (ortz) [inline README.md]: you have it twice

**Comment by ortz** [comment]
one comment on the documentation, besides that it looks good

**Review by ortz — APPROVED** [review]
please fix the additional line in the documentation, that's it...

File: README.md (hunk 1 above)
Comment (kaplanelad) [reply README.md]: Ops, my mistake. 
The line was removed.

**Pull request merged**